packages/react/src/components/accordion/Accordion.module.scss

```diff
@@ -76,6 +76,10 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+
+  & > div {
```

Comment:
I think it would be more future-proof to target the element using a plain class selector.  But to be honest, I did not come up with any good class name for this. Could it be just  ```.accordionHeading```? But we already have accordionHeadingContainer 😬 
This is more of a suggestion